corehq/util/metrics/prometheus.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+from prometheus_client import Counter as PCounter
+from prometheus_client import Gauge as PGauge
+from prometheus_client import Histogram as PHistogram
+
+from corehq.util.metrics.metrics import HqMetrics
+
+
+class PrometheusMetrics(HqMetrics):
+    """Prometheus Metrics Provider"""
+
+    def __init__(self):
+        self._metrics = {}
+
+    def _counter(self, name: str, value: float = 1, tags: dict = None, documentation: str = ''):
+        """See https://prometheus.io/docs/concepts/metric_types/#counter"""
+        self._get_metric(PCounter, name, tags, documentation).inc(value)
+
+    def _gauge(self, name: str, value: float, tags: dict = None, documentation: str = ''):
+        """See https://prometheus.io/docs/concepts/metric_types/#histogram"""
+        self._get_metric(PGauge, name, tags, documentation).set(value)
+
+    def _histogram(self, name: str, value: float, bucket_tag: str, buckets: List[int], bucket_unit: str = '',
+                  tags: dict = None, documentation: str = ''):
+        """
+        A cumulative histogram with a base metric name of <basename> exposes multiple time series
```

Comment:
You missed the one on this line
```suggestion
        A cumulative histogram with a base metric name of <name> exposes multiple time series
```

Reply:
Thanks: https://github.com/dimagi/commcare-hq/pull/26916